_scripts/app2-3.R

```diff
@@ -0,0 +1,53 @@
+{
+  
+  # Application 2-3 ----
+  
+  rm(list=ls());
+  options(show.error.locations = TRUE);
+  
+  # Part A ----
+  
+  vector1 <- seq(from = 45, to = 167, by = 1); # alt. = seq(45:167)
+  vector2 <- seq(from = 25, to = -7, by = -1); # alt. = seq(25:-7)
+  
+  # Part B ----
+  
+  vector3 <- seq(from = 17, to = 5, by = -3);
+  
+  for(j in vector3){
+    
+    cat("\n", j, "cubed power is", j^3, "\n");
+    cat("\n", j, "cubed root is", j^(1/3), "\n");
+    
+  }
+  
+  # Part C ----
+  
+  weatherData <- read.csv("_data/LansingWeather2.csv", stringsAsFactors = FALSE); 
+  
+  sampleDay <- c(13, 3, 5, 8, 7); # days to output high temp. and precipitation
+  
+  for(j in sampleDay){
```

Comment:
The alternate way to do this for()

for(j in 1:length(sampleDay))   # Again, j=1,2,3,4,5 because there are 5 values in sampleDay (i.e., its length is 5)
{
    # j=1,2,3,4,5 and is now an index for sampleDay: sampleDay[1] = 13, sampleDay[2] = 3...
    cat("\nOn day", sampleDay[j], "the high temperature and precipitation was",
        weatherData$highTemp[sampleDay[j]], "and", weatherData$precipitation[sampleDay[j]],
    ", respectively.", "\n");
  }